utils/awesome-client

```diff
@@ -12,12 +12,15 @@ if [ "$ISATTY" = 0 ]
 then
     # rlwrap provides readline functionality for "read", which is more enhanced
     # than bash's "read" itself.
-    RLWRAP=$(which rlwrap 2>/dev/null)
-    if [ "$RLWRAP" != "" ]
+    # It can be disabled/overridden using 'AWESOME_RLWRAP= awesome-client'.
+    if [ -z "${AWESOME_RLWRAP+x}" ]; then
```

Comment:
There is also `${FOO-value-if-unset}`, but that caused an issue with Vim's highlighting.  Also using an `if` here is less magical.